webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -212,6 +212,7 @@ const ContentViewVersions = ({ cvId, details }) => {
     [
       {
         title: __('Promote'),
+        ouiaId: `promote-version-${versionId}`,
```

Comment:
To be useful in testing, OUIA IDs should be static, e.g. the same OUIA ID always finds the same element on the page. The `${versionId}` makes it dynamic.

```suggestion
        ouiaId: `promote-version`,
```

If there will be more than one of these on the page, need to find a way to make it unique on the page without using database IDs. Maybe bring in a row index or something?

Reply:
Per offline discussion, should use row index here instead.